### ISSUE TITLE
What are the steps to install uWebsockets on Ubuntu 18.04? 

### ISSUE THREAD

**Issue by KabeerSadiq123** [SocketCluster/socketcluster#509]
Hi,
I am using uwebsocket for my project, but when I am installing it , sudo make install command is not working. So can anyone please mention the steps which i need to follow. I already install openSll and zlib . 
Thanks

**Comment by Charuru** [comment]
Make sure you're using node v9 or earlier, later versions of node does not support uws.

**Comment by SSANSH** [comment]
yes I am using node 12, what are steps to install uWebsockets and use then on socketcluster?
